File: temp_debug_parse.py
from pathlib import Path
from esbuild import transform_sync
import sys
file = Path('src/pages/NepaliPatro.jsx')
text = file.read_text('utf-8')
lines = text.splitlines()
for i in range(1, len(lines)+1):
    prefix = '\n'.join(lines[:i])
    try:
        transform_sync(prefix, loader='jsx')
    except Exception as e:
        print('error at line', i)
        print(str(e))
        sys.exit(0)
print('no error')

File: temp_debug_parse.js
const { transformSync } = require('esbuild');
const fs = require('fs');
const text = fs.readFileSync('src/pages/NepaliPatro.jsx', 'utf8');
const lines = text.split(/\r?\n/);
for (let i = 1; i <= lines.length; i++) {
  const prefix = lines.slice(0, i).join('\n');
  try {
    transformSync(prefix, { loader: 'jsx' });
  } catch (e) {
    console.log('error at line', i);
    console.log(e.message);
    process.exit(0);
  }
}
console.log('no error');
